ops/charm.py

```diff
@@ -297,6 +297,20 @@ def restore(self, snapshot):
             self.unit = None
 
 
+class RelationCreatedEvent(RelationEvent):
+    """Represents the `relation-created` hook from Juju.
+
+    It's triggered by `CharmBase.on[RELATION].relation_created`.
+
+    This is triggered whenever relations are created in the Juju model before relation-joined
```

Comment:
The sentence is not clear.. it's saying that this is fired "whenever relations are created before relation-join relations".

Reply:
Right, I suppose "whenever relations are created before relation-join **events**".

Reply:
https://github.com/canonical/operator/pull/218/commits/6e5916633b97995ca88a080597dff7c6e9ceff4c